docs/api.md

```diff
@@ -91,6 +91,51 @@ Specify `redirectTo` to set the path to redirect to after logging out. If this i
 <LogoutRoute redirectTo='/pathToRedirectTo' />
 ```
 
+## Contexts
+
+#### authenticated (bool)
+
+If you want to know if there's an authenticated account, then include the `authenticated` context.
+
+```javascript
+class AuthenticatedExample extends React.Component {
+  static contextTypes = {
+    authenticated: React.PropTypes.bool
+  };
+
+  render() {
+    return (
+      <p>
+        { this.context.authenticated ?
+          'Authenticated!' :
+          'Not authenticated!'
+        }
+      </p>
+    );
+  }
+}
+```
+
+#### account (object)
+
+If you want to retrieve the authenticated account, then include the `account` context. The value will be `undefined` if there's no authenticated account.
```

Comment:
I agree, and I tried to write it better. But I felt that it wasn't as clear. Do you have a suggestion? And yes, an unauthenticated account value is `undefined`. I.e. when there is no account.